modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -15,11 +15,17 @@ A form appears, where you can provide your contact information and create a user
 +
 . Enter the following:
 +
-* Username - Must be at least 3 characters and contain only the following: alphanumeric characters, dashes, or underscores.
-* Password - Must be at least 6 characters in length and contain only the following: upper and lower case characters, numbers, and non-alphanumeric symbols.
+* Username - Must be at least three characters and can contain only the following: +
+* Alphanumeric characters
+* Dashes, or 
+* Underscores
```

Comment:
Indent 19-21 under 18

Reply:
is it weird to have an "or" after Dashes but not after alphanumeric characters? Can we leave it out since this is a list?